controllers/platform/services/properties.go

```diff
@@ -160,7 +160,8 @@ func GenerateDataIndexWorkflowProperties(workflow *operatorapi.SonataFlow, platf
 	props := properties.NewProperties()
 	props.Set(constants.KogitoProcessDefinitionsEventsEnabled, "false")
 	props.Set(constants.KogitoProcessInstancesEventsEnabled, "false")
-	if workflow != nil && !profiles.IsDevProfile(workflow) && dataIndexEnabled(platform) {
+	di := NewDataIndexHandler(platform)
```

Comment:
another silly question... , imagine that in the current platform (the one existing in the worfklow namespace) we don't have for example a DataIndex configured, but instead we have a cluster platform that has. 
At which point are we taking the url of the cluster platform to configure this workflow?

Reply:
here -
https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/81e1e83002b8fa92a6bdf2e96ed040b7080ed8db/controllers/platform/services/properties.go#L168